examples/VLCB_4in4out/VLCB_4in4out.ino

```diff
@@ -143,8 +144,8 @@ void setupVLCB()
   // set module parameters
   VLCB::Parameters params(modconfig);
   params.setVersion(VER_MAJ, VER_MIN, VER_BETA);
-  params.setManufacturer(MANU_DEV);
   params.setModuleId(MODULE_ID);
+  params.setManufacturer(MANUFACTURER);
```

Comment:
Pleas set manufacturer before module ID. 
No functional difference. Just looks nicer as manufacturer has higher precedence.

Reply:
Done.  I've also modified in empty and 1in1out. You're right about precedence.  I'm sure that I thought about this and then got it wrong!  I blame chemo!